src/components/AddObsModal/AddObsButton.js

```diff
@@ -77,6 +77,7 @@ const AddObsButton = (): React.Node => {
       />
       <GradientButton
         sizeClassName="w-[69px] h-[69px] mb-[5px]"
+        onLongPress={!isAllAddObsOptionsMode && openModal}
```

Comment:
Have you considered always using `openModal` here? Then the prop wouldn't need to be optional.

Reply:
I considered this, but the way the issue is written we don't need the long press for the case where there's a plus button since the short press opens the modal. I imagine we'll end up tweaking this and maybe making the long and short press do the same thing for the plus button eventually.